broker-cli/commands/wallet.js

```diff
@@ -68,13 +68,28 @@ async function balance (args, opts, logger) {
       style: { head: ['gray'] }
     })
 
-    balances.forEach(({ symbol, uncommittedBalance, totalChannelBalance, totalPendingChannelBalance, uncommittedPendingBalance }) => {
+    balances.forEach((balance) => {
+      let {
+        symbol,
+        error = null,
+        totalChannelBalance,
+        totalPendingChannelBalance,
+        uncommittedBalance,
+        uncommittedPendingBalance
+      } = balance
+
+      totalChannelBalance = error ? 'Not Available'.yellow : totalChannelBalance.green
```

Comment:
We may use the error text at another time, however for right now to have the message fit nicely in the `wallet balance` table, we simply set the text to `Not Available`